templates/vue/src/components/TapestryMain/TapestryNode/index.vue

```diff
@@ -12,7 +12,10 @@
         'has-title': !node.hideTitle,
       }"
       :style="{
-        cursor: node.accessible || hasPermission('edit') ? 'pointer' : 'not-allowed',
+        cursor:
+          node.accessible || hasPermission('edit') || hasPermission('move')
+            ? 'pointer'
+            : 'not-allowed',
```

Comment:
Would be nice to have the `move` icon if it's only moving.
```suggestion
        cursor:
          node.accessible || hasPermission('edit') ? 'pointer' :
            (hasPermission('move') ? 'move' : 'not-allowed'),
```

Reply:
@wynnset Do we want the cursor to be `'move'` even on hover or just when we click and press down on a node? This change makes it so it's a move cursor is present on hover, even when hovering on the lightbox, add, and edit buttons.

Reply:
@shirleywbi on hover please. This will never happen when hovering on add / edit / view buttons because those buttons don't appear in the given conditional i.e. the negation of `node.accessible || hasPermission('edit')`.

Reply:
@wynnset I think there is one case for hovering on add. For instance, if a user is logged in, they can create draft nodes/submitted nodes even though they don't have permission. Also, with #937 allowing anyone to move the node (without saving it), I'm not sure what the expected behaviour should be in terms of the `move` cursor.

Reply:
Ok yes I guess we can just leave it as is then.